python-main.js

```diff
@@ -627,13 +628,14 @@ function web_editor(config) {
             doShare();
         });
         $("#command-help").click(function () {
```

Comment:
Currently we only toggle the view when the button is clicked, but since now it looks like a drop down menu then it should be closed when clicking anywhere outside the menu.